src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index].name().toLowerCase();

```

Comment:
Using `name().toLowerCase()` is not recommended as it introduces a dependency on the enum constant's name. It's better to have a specific method in the enum that returns the desired string representation.